agent/bench-scripts/tests/pbench-linpack/test-65.pre

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Setup a fake install of linpack but without the expected executable.
```

Comment:
s/without/with a stub for/ ?

Reply:
Fixed in https://github.com/ldoktor/pbench/pull/5.